advanced/Entity Matching with SDK-experimental demo.ipynb

```diff
@@ -307,9 +313,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Additional match_to items\n",
+    "### Add Additional key `feature_type`\n",
+    "\n",
+    "#### When to use the different feature type?\n",
+    "By default `feature_type` is set to \"simple\". You can try other feature_type when matching does not give you expected results. The options of feature_type are:\"simple\", \"bigram\", \"frequency-weighted-bigram\", \"bigram-extra-tokenizers\", \"bigram-combo\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### When to use `feature_type=simple`?\n",
+    "Simple is used by default for feature_type. This type is perfered to apply when one string is a substring of the other, and the rest characters are not appeared in the other string. For example \"BCDEF\" is a substring of \"A<u>BCDEF</u>G\", and the rest characters \"AG\" is not appeared in the string \"BCDEF\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sources = [\n",
```

Comment:
Maybe remove description and change id= 1 and id=2 in targets so that only the best option is the predicted one (show an example where simple works great?